odoo/addons/test_new_api/tests/test_properties.py

```diff
@@ -330,6 +330,7 @@ def test_properties_field_read_batch(self):
         ]
 
         self.env.invalidate_all()
+        self.env.cr.cache.pop('cache_exists', None)
```

Comment:
can't we just move `self.env.cr.cache.pop('cache_exists', None)` into `invalidate_all` ?

(people might not know that exists has a cache, and will just mechanically call `invalidate_all`, bonus, `invalidate_recordset` could remove the ids from the `exists` cache)